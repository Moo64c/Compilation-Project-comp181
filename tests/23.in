(map map (list 
             (lambda (x) (+ x x)) 
             (lambda (x) (- x))
             (lambda (x) (* x x)) 
             (lambda (x)  (/ x)))
       '((1 2) (3 4) (5 6) (7 8)))
