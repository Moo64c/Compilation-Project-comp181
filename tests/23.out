((2 4) (-3 -4) (25 36) (1/7 1/8))
